gcloud/logging/client.py

```diff
@@ -47,6 +49,28 @@ class Client(JSONClient):
     """
 
     _connection_class = Connection
+    _logging_api = _sinks_api = _metrics_api = None
+
+    @property
+    def logging_api(self):
+        """Helper for logging-related API calls."""
+        if self._logging_api is None:
+            self._logging_api = _LoggingAPI(self.connection)
```

Comment:
I'm just wary of the branch on every access. (I'm more sensitive to this because of the massive branching penalty on GPUs, but we aren't doing any super-computing here.) It's your call.